src/Hungary/Mews.Fiscalizations.Hungary/Utils/Client.cs

```diff
@@ -18,40 +19,46 @@ internal Client(HttpClient httpClient, NavEnvironment environment)
             Environment = environment;
         }
 
-        internal async Task<ResponseResult<TResult, TCode>> ProcessRequestAsync<TRequest, TDto, TResult, TCode>(string endpoint, TRequest request, Func<TDto, ResponseResult<TResult, TCode>> successFunc)
+        internal async Task<ResponseResult<TResult, TCode>> ProcessRequestAsync<TRequest, TDto, TResult, TCode>(
+            string endpoint,
+            TRequest request,
+            Func<TDto, string, string, ResponseResult<TResult, TCode>> successFunc)
             where TRequest : class
             where TDto : class
             where TResult : class
             where TCode : struct
         {
-            var httpResponse = await SendRequestAsync(endpoint, request);
-            return await DeserializeAsync(httpResponse, successFunc);
+            var parameters = new XmlSerializationParameters(namespaces: ServiceInfo.XmlNamespace.ToEnumerable());
+            var xmlRequest = XmlSerializer.Serialize(request, parameters);
```

Comment:
I'd create the variable requestContent here, so that we don't need to do the `requestXml.OuterXml` multiple times.